src/components/Scope/index.js

```diff
@@ -60,8 +68,6 @@ export const Scope = ({ children, filterPlaceholder }) => {
 				show={show}
 				reset={reset}
 				getScope={getScope}
-				nodeState={nodeState}
-				defaultNodeState={defaultNodeState}
```

Comment:
Reinstate this - that way TreeView handles the default. (The nodeState prop is not received by Selector, so that can stay gone.)